BUILD.bazel

```diff
@@ -0,0 +1,56 @@
+licenses(["notice"])
+
+exports_files(["LICENSE"])
```

Comment:
Is this needed?

Reply:
AFAIK bazel only has _plans_ to support license files as part of the build, but since the repo provides it I am following (what I believe to be) an accepted convention of exporting it.
GoogleTest does this, [for instance](https://github.com/google/googletest/blob/master/BUILD.bazel).